rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -190,6 +190,7 @@ PYBIND11_MODULE(_transport, m) {
     &PlayOptions::setTopicQoSProfileOverrides)
   .def_readwrite("loop", &PlayOptions::loop)
   .def_readwrite("topic_remapping_options", &PlayOptions::topic_remapping_options)
+  .def_readwrite("clock_publish_frequency", &PlayOptions::clock_publish_frequency)
```

Comment:
The reason I prefer frequency is that we always deal in Hz - the domains we deal with do not require kHz, MHz, GHz, as the unit. Whereas when you specify a period, you _always_ have to very carefully specify the unit. I think it's harder to use in general because you have to read the documentation to understand what units you are working in.

Reply:
I'll also add that - for me - thinking about frequency is easier. I want to say "publish this message 20 times per second. I don't want to have to do mental math and say ... ok that's how many milliseconds/nanoseconds/microsedons? is that 50? maybe, i'll pull up the calculator". Absent a psychological study proving that one is easier than the other for people across different native language groups to use, I'm going to skip this comment, though I do appreciate you thinking carefully about this. After all, it's `ros2 topic hz`, not `ros2 topic period`.